### ISSUE TITLE
compthinking1

### ISSUE THREAD

**Issue by syamamo5** [syamamo5/compthinking1#1]
syamamo5.github.io/compthinking1

Are we supposed to update our computational project organization notes?

**Comment by syamamo5** [comment]
@FlukeAndFeather
Is this how we tag people?

**Comment by FlukeAndFeather** [comment]
Yes, you tagged me correctly. Thanks for submitting your assessment!

_Feedback_

**Q1** Correct!

**Q2** Your answer is technically correct, but the question was trying to get at something different. The purpose of "explain in plain language" questions is to have you paraphrase or summarize what the code is doing as a whole, rather than a line-by-line description. Research shows the ability to succinctly explain what code does is important for writing good code, so that's what I'm trying to get you to do here. By the end of the quarter, I want you to be able to look at something like this code chunk and say "this finds how many birds' masses are outliers".

**Q3** The idea of this question was to change the input to get a new output. You start with this code:

```r
quad_counts <- c(2L, 19L, 4L, 5L, 5L, 12L, 0L, 7L)
valid_quads <- c(1, 2, 3, 5, 7)
quad_counts2 <- quad_counts[valid_quads]
median_count <- median(quad_counts2)
```

Your output is the value of `median_count`. When you run the code unmodified, what's the value of `median_count`? How could you change the input, `quad_counts`, so `median_count` ends up as 5 instead?



Great work!
